### PR TITLE
rbd cephci suite fix for nautilus and py2 to py3 conversion

### PR DESCRIPTION
**Pull request by Gopi-Patta**
rbd cephci suite fix for both luminous & nautilus and py2 to py3 conversion.

Signed-off-by: gpatta@redhat.com

### CODE REVIEW

**Comment by Gopi-Patta** [comment]
rhcs3 logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1591254076377/test_83572723_0.log
rhcs4 logs:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1590931855215/test_83572723_0.log

**Comment by XDsubhash** [comment]
@Gopi-Patta this PR fixes this one script right?

File: rbd/system/cli/rbd_cli_import_export_diff.py [hunk 1]
```diff
@@ -41,16 +42,11 @@ def exec_cmd(args):
     if k_m:
         combinations = cli.generate_combinations('image_size', 'object_size',
                                                  'image_format', 'data_pool')
-        invalid = [val for val in combinations
-                   if (cli.search_param_val('--image-format', val) != 0 and
-                       cli.search_param_val('--image-format', val)
```
Comment (XDsubhash) [inline rbd/system/cli/rbd_cli_import_export_diff.py:46]: @Gopi-Patta what was the problem here wrt nautilus? any rbd cli changes

Comment (rakeshgm) [reply rbd/system/cli/rbd_cli_import_export_diff.py:46]: @Gopi-Patta ??

Comment (Gopi-Patta) [reply rbd/system/cli/rbd_cli_import_export_diff.py:46]: @XDsubhash  removed as image-format 1 is not supporting now.

Comment (XDsubhash) [reply rbd/system/cli/rbd_cli_import_export_diff.py:46]: @Gopi-Patta what abt in luminous ? image-format is working fine right?

Comment (rakeshgm) [reply rbd/system/cli/rbd_cli_import_export_diff.py:46]: @Gopi-Patta , I accidentally merged this PR. please check on luminous as well. there is no separate branch for luminous as well.

Comment (Gopi-Patta) [reply rbd/system/cli/rbd_cli_import_export_diff.py:46]: @XDsubhash Image-format 1 will work in luminous with warning. Please ignore this PR as i will send one more by including image-format 1 when it is luminous.

**Pull request merged**